src/runtime_src/core/pcie/driver/linux/xocl/subdev/ddr_srsr.c

```diff
@@ -126,7 +112,7 @@ static int srsr_save_calib(struct platform_device *pdev)
 	}
 	xocl_dr_reg_write32(xdev, CTRL_BIT_SREF_REQ | CTRL_BIT_XSDB_SELECT, xocl_ddr_srsr->base+REG_CTRL_OFFSET);
 
-	for ( i = 0; i < 4; ++i) {
+	for (i = 0; i < 4; ++i) {
```

Comment:
need a macro for 4. what is 4 here?

Reply:
The calibration cache size is up to 2901 words(so far), means ~12KB, and XSDB size is 4KB.
So we have to read the calibration data from XSDB at least 3 times.

Let's make it to 4, then we have 16kB calibration cache.